### DISCUSSION TITLE
Contacts fork already maintained elsewhere

### DISCUSSION THREAD

**Discussion by Cwpute** [FossifyOrg/General-Discussion#658] [Ideas]
We are several users now using a fork of Simple-Contacts made by @stephanritsther,  which has seen several improvments made by them while Simple-Contacts was still not sold.
Since there was a qaestion to maybe not continue the development of some of the original apps, i'd suggest leaving this one behind since a good fork is already maintained and followed :)

**Comment by inson1** [comment]
https://github.com/stephanritscher/Simple-Contacts

**Comment by vertigo220** [comment]
Doesn't look like that fork is intended to be maintained, but rather is just to add a single feature. On the main page, it actually links to this project, and there's talk in its issues about merging with this one.
